### ISSUE TITLE
Foreign Keys icon

### ISSUE THREAD

**Issue by mickeyyde** [qwtel/sqlite-viewer-vscode#37]
Foreign key icon does not show if the foreign key name is not the same as the primary key name
See example:

```
CREATE TABLE juncMEP(
    ip_address VARCHAR(15), 
    mep_id INT,
    peer_mep_id INT,
    port INT,
    MEG VARCHAR(255),
    FOREIGN KEY (ip_address) REFERENCES VLAN100_ADMIN(ip_address),

    FOREIGN KEY (mep_id) REFERENCES MEP(mep_id),
    FOREIGN KEY (peer_mep_id) REFERENCES MEP(mep_id)

);
```

![Capture](https://github.com/qwtel/sqlite-viewer-vscode/assets/92148246/9ac55ceb-e728-4657-a64e-f045a2346159)

As you can see "peer_mep_id" do not have the icon, there is also a weird behavior if you point the "mep_id" icon
Below the MEP table if you need

```
CREATE TABLE MEP (
    mep_id INT UNIQUE PRIMARY KEY,
    vid INT,
    domain_name VARCHAR(255),
    mac VARCHAR(17)
);
```

**Comment by qwtel** [comment]
this will be fixed in an upcoming release

**Comment by qwtel** [comment]
This should be fixed in v0.4.11
